crates/move-prover-lean-backend/src/generator_options.rs

```diff
@@ -0,0 +1,49 @@
+use log::LevelFilter;
+use move_stackless_bytecode::options::ProverOptions;
+use serde::{Deserialize, Serialize};
+use crate::lean_backend::options::LeanOptions;
+
+/// Represents options provided to the tool. Most of those options are configured via a toml
+/// source; some over the command line flags.
+///
+/// NOTE: any fields carrying structured data must appear at the end for making
+/// toml printing work. When changing this config, use `mvp --print-config` to
+/// verify this works.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct Options {
+    /// The path to the boogie output which represents the verification problem.
```

Comment:
If we're keeping the boogie copy-pasta, it's worth doing a pass to replace or delete these comments. 

It might be good to comment files at the top that are entirely copied from the Boogie without changes.